lib/rubycritic/generators/html/assets/javascripts/application.js

```diff
@@ -1,107 +1,240 @@
 prettyPrint();
 
+// sidebar navigation
+$(function() {
+  var loc = window.location.href; // returns the full URL
+  if(/overview/.test(loc)) {
+    $('.overview-nav').addClass('active');
+  }
+  else if(/code_index/.test(loc)) {
+    $('.code-index-nav').addClass('active');
+  }
+  else if(/smells_index/.test(loc)) {
+    $('.smells-index-nav').addClass('active');
+  }
+});
+
+var turbulenceData = turbulenceData || [];
+var COLOR = {
+  'A' : '#00B50E',
+  'B': '#53D700',
+  'C': '#FDF400',
+  'D': '#FF6C00',
+  'F': '#C40009'
+};
+$("#churn-vs-complexity-graph-container").highcharts({
+  chart: {
+    type: "scatter",
+    zoomType: "xy"
+  },
+  title: {
+    text: "Churn vs Complexity"
+  },
+  xAxis: {
+    title: {
+      enabled: true,
+      text: "Churn"
+    },
+    floor: 0,
+    startOnTick: true,
+    endOnTick: true
+  },
+  yAxis: {
+    title: {
+      text: "Complexity"
+    },
+    floor: 0,
+    startOnTick: true,
+    endOnTick: true
+  },
+  plotOptions: {
+    series: {
+      turboThreshold: 1500
+    },
+    scatter: {
+      marker: {
+        radius: 5,
+        states: {
+          hover: {
+            enabled: true,
+            lineColor: "rgb(100,100,100)"
+          }
+        }
+      },
+      tooltip: {
+        headerFormat: "<b>{point.key}</b><br>",
+        pointFormat: "Committed {point.x} times, with Flog score of {point.y}"
+      }
+    }
+  },
+  series: [{
+    showInLegend: false,
+    color: "steelblue",
+    data: getMappedTurbulenceData()
+  }]
+});
+
+function getMappedTurbulenceData(){
+  var dataWithColorInformation = turbulenceData.map(function(data){
+    data.color = COLOR[data.rating];
+    return data;
+  });
+  return dataWithColorInformation;
+};
+
+$(function() {
+  $("#gpa-chart").highcharts(
+    {
+      chart: {
+        type: 'pie'
+      },
+      title: {
+        text: 'Grade Point Average'
+      },
+      plotOptions: {
+        pie: {
+          shadow: false
+        }
+      },
+      tooltip: {
+        formatter: function() {
+          return '<b>'+ this.point.name +'</b>: '+ this.y +' %';
+        }
+      },
+      series: [{
+        name: 'Browsers',
+        data: getGpaData(),
+        size: '90%',
+        innerSize: '65%',
+        showInLegend:true,
+        dataLabels: {
+          enabled: false
+        }
+      }]
+    },
+    function(chart) { // on complete
+      var textX = chart.plotLeft + (chart.plotWidth  * 0.5);
+      var textY = chart.plotTop  + (chart.plotHeight * 0.5);
+      var span = '<span id="pieChartInfoText" style="position:absolute; text-align:center;">';
+      span += '<span style="font-size: 32px">'+((score/100)*4).toFixed(2)+'</span><br>';
```

Comment:
Although smaller numbers look better, I think we should still rely on our _0 to 100_ score system (https://github.com/whitesmith/rubycritic/blob/master/docs/core-metrics.md#score)